doc/db.md

```diff
@@ -112,3 +112,12 @@ Create a new database on the same server as `dbname` owned by the `dbname` owner
 ex:
 gen3 db restore fence fence_20190101.backup
 ```
+
+### gen3 db encrypt account profile
+
+Used to encrypt the rds databases. You should use the account/profile from gen3 workon. There is also an extra argument you can add to specify the directory to hold the db snapshots. This should be used for larger db's where the root volume is not large enough to store the snapshots.
```

Comment:
Should document this as "profile workspace" to be consistent with the workon documentation - maybe also mention that the workspace should be the one that deploys the VPC and RDS, and that the command will modify the state file, creds in the current namespace, for all 3 db's, and whether the user will need to deallocate the old RDS instances ...